src/Datagrid/OrderByToSelectWalker.php

```diff
@@ -30,6 +34,16 @@
  */
 final class OrderByToSelectWalker extends TreeWalkerAdapter
 {
+    public function __construct($query, $parserResult, array $queryComponents)
```

Comment:
IMO, `Doctrine\ORM\AbstractQuery` and `Doctrine\ORM\Query\ParserResult` should be used for the type declarations in argument 1 and 2.

Reply:
There is no type used in the parent constructor.

And the OrderByToSelectWalker class is deprecated.

Reply:
I saw that, but the the properties that these arguments set have these type hints declared as docblocks.

Reply:
The interface TreeWalker has no type in the constructor, so I cannot add them.